scripts/zones/Waughroon_Shrine/bcnms/beyond_infinity.lua

```diff
@@ -0,0 +1,46 @@
+
+
```

Comment:
Get rid of these extra spaces

Reply:
fixed. sorry i was copy and pasting left and right cuz of the rebase errors that had to create a new branch lol.